src/api_server/src/attest/amber.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2023 by Intel.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::Attest;
+use anyhow::*;
+use as_types::AttestationResults;
+use async_trait::async_trait;
+use jsonwebtoken::{decode, decode_header, jwk, DecodingKey, Validation};
+use kbs_types::{Attestation, Tee};
+use reqwest::header::{ACCEPT, CONTENT_TYPE};
+use serde::{Deserialize, Serialize};
+use std::fs::File;
+use std::io::BufReader;
+
+#[derive(Deserialize, Debug)]
+struct TdxEvidence {
+    _cc_eventlog: Option<String>,
+    quote: String,
+}
+
+#[derive(Serialize, Debug)]
+struct AttestReqData {
+    quote: String,
+}
+
+#[derive(Deserialize, Debug)]
+struct AttestRespData {
+    token: String,
+}
+
+#[derive(Deserialize, Debug)]
+struct Claims {
+    amber_unmatched_policy_ids: Option<Vec<serde_json::Value>>,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct AmberConfig {
+    pub base_url: String,
+    pub api_key: String,
+    pub certs_file: String,
+    pub check_policy: Option<bool>,
+}
+
+pub struct Amber {
+    config: AmberConfig,
+    jwks: jwk::JwkSet,
```

Comment:
I think these are the certificate, JWK-formatted? If so, could we rename that field to `certificates` or `keys`?

Reply:
Thanks your comments, I will modify.